tests/test_copilot_cli.py

```diff
@@ -115,6 +115,48 @@ def test_exit_if_no_local_copilot_environments(self, fakefs):
         assert result.exit_code == 1
         assert result.output == "No environments found in ./copilot/environments; exiting\n"
 
+    def test_env_addons_parameters_file_is_written_with_redis_storage_type(self, fakefs):
```

Comment:
Minor one, you could use `@pytest.mark.parametrize` and use a single test.

Reply:
I think it should be written with Postgres storage types too.

Reply:
That is a good point... I will attempt this.

Reply:
@WillGibson It is, but just not tested, as it would be duplication... but Lyndon's point around parametrize would get us around this.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/107/commits/e371ffed0a57da1bd0d9ca7d3a6825e1324da396.